docker-compose.yaml

```diff
@@ -14,8 +11,26 @@ services:
       - VELOX_ROLE=${VELOX_ROLE}
       - VELOX_SERVER_URL=${VELOX_SERVER_URL}
       - VELOX_FRONTEND_HOSTNAME=${VELOX_FRONTEND_HOSTNAME}
+      - VELOX_API_HOSTNAME=velociraptor
     ports:
       - "8000:8000"
       - "8001:8001"
       - "8889:8889"
-    restart: unless-stopped
+    networks:
+      - velo
+
+  velociraptor_minion:
```

Comment:
We might want to make the usage of a client container optional. We should consider using something like the following:

```
container_name: velociraptor_minion
profiles: 
      - minion
```

Then `docker-compose` can be invoked like:

`docker-compose --profile minion up`